addons/website_event/static/src/snippets/s_events/000.scss

```diff
@@ -0,0 +1,93 @@
+.s_dynamic {
+    .s_events_event_title {
+        font-weight: $headings-font-weight;
+        line-height: 1;
+    }
+
+    .s_events_event_subtitle {
+        font-size: 1em;
+    }
+
+    &.s_event_event_big_picture {
+        .s_events_event {
+            min-height: 150px;
+
+            figcaption {
+                position: relative;
+                justify-content: center;
+                pointer-events: auto;
+            }
+
+            .s_events_event_cover {
+                min-height: 100%;
+
+                .o_record_cover_container {
+                    top: 0;
+                }
+            }
+
+            .s_events_event_title {
+                @include font-size($h3-font-size);
+                margin-bottom: 0.5em;
+                word-spacing: -0.15em;
+            }
+        }
+
+        .row {
+            align-items: stretch;
+            .s_events_event_subtitle {
+                margin: 0;
+            }
+        }
+    }
+
+    &.s_event_event_card {
+        .card {
+            height: 100%;
+
+            .s_events_event_cover {
+                height: 150px;
+
+                .o_record_cover_container {
+                    background-color: transparent !important;
+
+                    .o_record_cover_image {
+                        @extend .card-img-top;
+                        height: inherit;
+                    }
+                }
+
+                .s_events_event_date {
+                    top: $card-spacer-x;
+                    right: $card-spacer-x;
+                    display: flex;
+                    flex-direction: column;
+                    justify-content: center;
+                    @include size(4rem);
+                    border-radius: 50%;
+                    text-align: center;
+
+                    .s_events_event_month {
+                        font-size: 0.75rem;
+                        font-weight: $font-weight-bold;
+                        text-transform: uppercase;
+                    }
+                    .s_events_event_day {
+                        font-size: 1.125rem;
+                        font-weight: 300;
+                        line_height: 1;
+                    }
+                }
+            }
+
+            a {
+                color: black;
+            }
```

Comment:
This is more than probably wrong :wink: Colors should never be hardcoded, they should depend on the theme. At minimum I guess what you want is to use the normal text color? (so use the related variable). At best you want a contrasting color in comparison with the background... in that case `color-yiq`?